ios/Runner/Info.plist

```diff
@@ -43,8 +43,6 @@
 	<true/>
 	<key>NSCameraUsageDescription</key>
 	<string>Camera permission is required for qr code scanning.</string>
-	<key>NSPhotoLibraryUsageDescription</key>
```

Comment:
is it possible to load a cert from a picture you took earlier of the qr code?

Reply:
When this was included we didn't have an option to ignore the photo library

Reply:
so i'm guessing the answer is no then?

Reply:
I'm sure it could be technically possible but its not a flow I am particularly interested in preserving